src/dcd/server/main.d

```diff
@@ -197,7 +197,19 @@ int runServer(string[] args)
 
 	serverLoop: while (true)
 	{
-		auto s = socket.accept();
+		Socket s;
+		try
+		{
+			s = socket.accept();
+		}
+		catch (SocketOSException e)
```

Comment:
real accept errors throw SocketAcceptException, so we can safely catch SocketOSException here